### PR TITLE
Review of Docker Compose sample

### PR DESCRIPTION
**Pull request by kbaley**
Relates to #5150

### CODE REVIEW

File: samples/platformtools-docker-compose/sample.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ Gracefully stops and removes the containers and the configured volumes.
 
 ### Updating
 
-The docker images are following semver. Meaning, breaking changes are only introduced in new majors. Releases are pushed as `major.minor.patch` and it is safe to follow a `major` tag to ensure updates.
+The Docker images follow [semantic versioning](https://semver.org/). In other words, breaking changes are introduced only in new major versions. Releases are pushed as `major.minor.patch` and it is safe to follow a `major` tag to ensure updates.
```
Comment (kbaley) [inline samples/platformtools-docker-compose/sample.md:77]: Do we have a concrete definition of what a "breaking change" is for ServiceControl and ServicePulse?

Also, I'm fuzzy on what it means to follow a major tag to ensure updates.

**Comment by kbaley** [comment]
@WilliamBZA @dvdstelt @ramonsmits I pinged you for a review since you were involved in the initial creation of this. You can look through the whole thing but I'm mostly interested in [this section](https://github.com/Particular/docs.particular.net/pull/5188/files#r561117453).

File: samples/platformtools-docker-compose/sample.md (hunk 1 above)
Comment (WilliamBZA) [reply samples/platformtools-docker-compose/sample.md:77]: We have this: https://github.com/Particular/Platform/blob/main/principles/versioning.md#platform-components-eg-servicecontrol-servicepulse-and-serviceinsight

Comment (WilliamBZA) [reply samples/platformtools-docker-compose/sample.md:77]: > Also, I'm fuzzy on what it means to follow a major tag to ensure updates.

It means you can have your docker images set to for e.g. `particular/servicecontrol.rabbitmq.conventional.monitoring-windows:4` and your docker images will pull the latest version `4.x.x` but if version `5.0.0` is released it won't automatically upgrade to version 5.

**Review by WilliamBZA — APPROVED** [review]

**Review by dvdstelt — APPROVED** [review]

File: samples/platformtools-docker-compose/sample.md (hunk 1 above)
Comment (kbaley) [reply samples/platformtools-docker-compose/sample.md:77]: > We have this: [Particular/Platform@`main`/principles/versioning.md#platform-components-eg-servicecontrol-servicepulse-and-serviceinsight](https://github.com/Particular/Platform/blob/main/principles/versioning.md#platform-components-eg-servicecontrol-servicepulse-and-serviceinsight)

I'm on the fence on whether we should include some _very_ short version of that document here to help people reading this determine what a breaking change is. I'm leaning toward waiting until someone asks just for the sake of keeping this simpler.

**Pull request merged**